app/test_main.py

```diff
@@ -1 +1,57 @@
-# write your code here
+from unittest import mock
+from typing import Callable
+
+import pytest
+
+from app.main import can_access_google_page
+
+
+TEST_URL = "https://github.com/Kirontiko/"
```

Comment:
could u pass it directly to the func ? on line 57)

Reply:
Oh, sorry, I understood wrong